.github/workflows/osd-increment-plugin-versions.yml

```diff
@@ -126,7 +126,7 @@ jobs:
                   repo: "${{ matrix.entry.repo }}",
                   name: labelName,
                   color: randomColor,
-                  description: 'Label description'
+                  description: "Issues targeting release ${{labelName}}"
```

Comment:
This label can be even added to a PR as well @gaiksaya, hence I have left the description with the default.

Reply:
PRs are issues as well technically :p 
But yeah I can keep it as ${{labelName}}. Let me know

Reply:
Thanks! By default it leaves blank but because of this it is adding as `Label description`